src/pubd/repository.rs

```diff
@@ -1311,11 +1328,14 @@ impl RepoStats {
 
 impl fmt::Display for RepoStats {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        writeln!(f, "RSYNC BASE:        {}", self.sia_base)?;
+        writeln!(f, "RRDP NOTIFICATION: {}", self.rrdp_notification_uri)?;
```

Comment:
Why not "URI" ?

Reply:
I wanted it to be similar to the CLI, but thanks for raising this, because it actually wasn't..

Changed it now to report the rsync and rrdp base URIs as they are also used when initialising the server and renamed the fields appropriately.